checks.d/process.py

```diff
@@ -241,8 +245,34 @@ def get_process_state(self, name, pids):
             st['r_bytes'].append(ioinfo.get('read_bytes'))
             st['w_bytes'].append(ioinfo.get('write_bytes'))
 
+            pagefault_stats = self.get_pagefault_stats(pid)
+            if pagefault_stats is not None:
+                (minflt, cminflt, majflt, cmajflt) = pagefault_stats
+                st['minflt'].append(minflt)
+                st['cminflt'].append(cminflt)
+                st['majflt'].append(majflt)
+                st['cmajflt'].append(cmajflt)
```

Comment:
I think that `if pagefault_stats is None`, we should append `None`, otherwise I fear it might mess up the map for PID -> stats for subsequent `pids`. See `st['real']` above.